compiler/src/iree/compiler/Codegen/LLVMGPU/ROCDLLowerExecutableTarget.cpp

```diff
@@ -70,7 +70,7 @@ class ROCDLLowerExecutableTargetPass final
       addGPUBaseLoweringPassPipeline(pipeline);
       break;
     case CodeGenPipeline::LLVMGPUWarpReduction:
-      addGPUWarpReductionPassPipeline(pipeline);
+      addGPUWarpReductionPassPipeline(pipeline, true);
```

Comment:
It would help to name this parameter here:
```suggestion
      addGPUWarpReductionPassPipeline(pipeline, /*forROCDL=*/true);
```